Source/Widgets/Viewer/viewerDragDropMixin.js

```diff
@@ -150,6 +152,20 @@ define([
             },
 
             /**
+             * Gets or sets a value indicating if the camera should fly to the data source after it is loaded.
+             * @memberof viewerDragDropMixin.prototype
+             * @type {Boolean}
+             */
+            flyToOnDrop : {
+                get : function() {
+                    return flyToOnDrop;
+                },
+                set : function(value) {
+                    flyToOnDrop = value;
```

Comment:
So I had a suspicion the reason was jsDoc and I was right.  For some reason assigning the value directly to viewer and documenting it causes it not to show up in the doc.  So we'll just leave it the way it is for now.